### ISSUE TITLE
Unavailable sensors will not update until HA restart

### ISSUE THREAD

**Issue by dexonter** [tofuSCHNITZEL/wienerlinien#15]
**Version of the custom_component**

1.3.0 beta
**Describe the bug**
If there is no data at the time of HA restart (during the night) - the sensors will never update once the morning comes
If the HA is restarted during the day, the sensors survive the night and show data once again the next morning
**log**

```
Can be provided if you tell me how...
```

**Comment by mr-p666** [comment]
Hej, 
is there any chance to get this annoying bug fixed? 
I would really appreciate this. 
Thx a lot! 👍🏼

**Comment by tofuSCHNITZEL** [comment]
Yes I know about it, but I don't know how to fix it currently. the whole logic of this integration is not "compatible" with the fact that sometimes stops are missing from the data just because its e.g. night and the lines don't run

**Comment by mr-p666** [comment]
Thx a lot for your answer!
Is it possible to insert a dummy stop when no data is received or to set a dummy one by default and only overwrite it when data is received?
